srcpkgs/anydesk/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'anydesk'
+pkgname=anydesk
+version=4.0.1
+revision=1
+archs="i686 x86_64"
+wrksrc=${pkgname}-${version}
+short_desc="Fast remote desktop application"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="Free for personal use"
+#Terms and conditions:https://anydesk.com/en/terms
+homepage="https://anydesk.com/remote-desktop"
+nopie=yes
+restricted=yes
+
```

Comment:
Shouldn't it be in `repository=nonfree`?

Reply:
No, see the answer in the package request.